src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/ExpectedOutputs/SARIF1010.RuleIdMustBeConsistent_Valid.sarif

```diff
@@ -47,7 +77,54 @@
           }
         }
       ],
-      "results": [],
+      "results": [
```

Comment:
>results [](start = 7, length = 7)

The "expected output" from a "valid" test shouldn't have any results. Something must be wrong in the logic.

Reply:
The problem here is that the rule 1010 and 2004 overlap in one scenario. Should i remove that scenario (when result.ruleId exists and result.rule.Id exists)

---
In reply to: [454707521](https://github.com/microsoft/sarif-sdk/pull/1995#discussion_r454707521) [](ancestors = 454707521)

Reply:
No, just disable 2004 for this test.

---
In reply to: [454713672](https://github.com/microsoft/sarif-sdk/pull/1995#discussion_r454713672) [](ancestors = 454713672,454707521)